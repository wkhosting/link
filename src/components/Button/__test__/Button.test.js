import Button from '../Button';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<Button />', () => {
  test('renders without exploding', () => {
    const root = document.getElementById('root');
    if (root) {
      root.render(<Button />);
    }
  });
  test('<Button /> snapshot', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
