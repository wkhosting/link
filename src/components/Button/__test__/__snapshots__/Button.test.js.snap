// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> <Button /> snapshot 1`] = `
<a
  className="button button-undefined"
  onClick={[Function]}
  rel="noopener noreferrer"
  target="_blank"
/>
`;
